.gitignore

```diff
@@ -1,5 +1,6 @@
 .env
 .tmp-id
+.idea/
```

Comment:
we don't need to add IDE specific ignores here. you can add them to you local `.git/info/exclude`